firefox_desktop/views/history_sidebar_clients_daily.view.lkml

```diff
@@ -0,0 +1,86 @@
+view: history_sidebar_clients_daily {
+
+  sql_table_name: `moz-fx-data-shared-prod`.telemetry_derived.clients_daily_joined_v1 ;;
```

Comment:
is there a reason you are referencing the _v1 version of this table? we usually use the views where possible, e.g. `mozdata.telemetry.clients_daily`

Also, we have in the past preferred the extension of things that are already in looker-hub, [and it looks like there is already a view for clients_daily_joined here](https://github.com/mozilla/looker-hub/blob/main/firefox_desktop/views/clients_daily_joined_table.view.lkml).

[Here is an example of how that kind of extension can work](https://github.com/mozilla/looker-spoke-default/blob/main/firefox_desktop/views/clients_daily_search.view.lkml#L1-L4). 

[Here are docs on extending views](https://cloud.google.com/looker/docs/reference/param-view-extends)

I could also be missing something here - is there is a reason why you need to have it the way you do?

Reply:
I didn't want all of the fields brought in from the `clients_daily_joined_table` view, which is why I was referencing the SQL table. I've changed it to extend the view, though -- having those fields in the future is probably worthwhile anyways.

Reply:
OK I see - there are a lot of fields in the source table, and it seems like you don't need too many of them. 

[You can always hide the ones you don't want](https://community.looker.com/lookml-5/hiding-fields-from-an-extended-view-2755?postid=7310#post7310) - all you have to do is set the `hidden` field to `yes` for the fields that you don't want to include from the source view. 

Alternatively, if you feel that makes the lookml too bloated (many lines just to hide each field; I don't think there's another way) then I would be OK with you just referencing the table directly (though the view, not the materialized version like you originally had).

I'll leave it up to you.